<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APP_AES</class>
 <widget class="QWidget" name="APP_AES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AES encrypt &lt;-&gt; decrypt</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>30</y>
     <width>383</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the 256(128)-bit key , for example: 02acd5b72b893296ba67.......</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_256bit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>256 bit</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">key_bit</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_128bit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>128 bit</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">key_bit</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_CBC">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CBC</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">CBC_ECB</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_ECB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ECB</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">CBC_ECB</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IV">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialization vector for CBC (128 bit)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gen_key_iv">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate key,  iv</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Copy key &amp; iv to clipboard</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="icon">
    <iconset resource="aes.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="aes.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="CBC_ECB"/>
  <buttongroup name="key_bit"/>
 </buttongroups>
</ui>
